http/rest/handler.go

```diff
@@ -0,0 +1,107 @@
+package rest
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"raccoon/config"
+	"raccoon/logger"
+	"raccoon/metrics"
+	"raccoon/pkg/collection"
+	"raccoon/pkg/deserialization"
+	"raccoon/pkg/identification"
+	pb "raccoon/pkg/proto"
+	"raccoon/pkg/serialization"
+	"time"
+)
+
+type Handler struct{}
+
+func (h *Handler) GetRESTAPIHandler(c collection.Collector) http.HandlerFunc {
+	return func(rw http.ResponseWriter, r *http.Request) {
+		contentType := r.Header.Get("Content-Type")
+		rw.Header().Set("Content-Type", contentType)
+		var group string
+		group = r.Header.Get(config.ServerWs.ConnGroupHeader)
+		if group == "" {
+			group = config.ServerWs.ConnGroupDefault
+		}
+		identifier := identification.Identifier{
+			ID:    r.Header.Get(config.ServerWs.ConnIDHeader),
+			Group: group,
+		}
+
+		d, s := h.getDeserializerSerializer(contentType)
```

Comment:
This line could move to 24 and in that way you'd validate and fail fast (for a bad request) even before the identifier is constructed. wdyt?

Reply:
Done